AGENTS.md

```
@@ -23,13 +29,28 @@ npm run typecheck
+
+When only specific tests are relevant (e.g., during reviews):
+
+```bash
+npm test -- src/webview-src/__tests__/event.handlers.test.tsx
+```
+
+
 # Launch extension in VS Code
 # Press F5 in VS Code, or:
 code "$(pwd)" --extensionDevelopmentPath="$(pwd)"
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)